app/controllers/application_controller.rb

```diff
@@ -132,7 +132,12 @@ def session_remember_me
   # is not replayable forever
   SESSION_TTL = 3.hours
   def validate_session_timestamp
-    if session[:user_id] && session[:ttl] && session[:ttl] < SESSION_TTL.ago
+    session_ttl = if session[:ttl].is_a?(String)
+      Time.zone.parse(session[:ttl]) # for Ruby 1.9
```

Comment:
That seemed to fails consistently on 1.9 specs with the new`config.action_dispatch.cookies_serializer` setting but without that code change